PIL/ImageShow.py

```diff
@@ -69,8 +69,9 @@ def show(self, image, **options):
             # FIXME: auto-contrast if max() > 255?
         else:
             base = Image.getmodebase(image.mode)
-        if base != image.mode and image.mode != "1":
+        if base != image.mode and image.mode != "1" and image.mode != "RGBA":
             image = image.convert(base)
```

Comment:
`'start "Pillow" /WAIT "%s" '`

No, it's not a built-in viewer, `"Pillow"` is a required title:

```
Syntax
      START "title" [/D path] [options] "command" [parameters]

Key:
   title       Text for the CMD window title bar (required.)
   path        Starting directory.
   command     The command, batch file or executable program to run.
   parameters  The parameters passed to the command.
```
https://ss64.com/nt/start.html

It just tells Windows to open an image file by giving the path to the file, and it'll open it in whatever program is associated with that file type. Usually IrfanView for me, but we need it in a format that will open in something default like MS Paint.